Libraries/ReactIOS/YellowBox.js

```diff
@@ -186,6 +225,7 @@ class YellowBox extends React.Component {
     super(props, context);
     this.state = {
       inspecting: null,
+      stacktraceVisible: false,
       warningMap: _warningMap,
     };
     this.dismissWarning = warning => {
```

Comment:
Lint runs well on the first commit but it does not delete irrelevant messages after they are fixed.